singer/src/main/java/com/pinterest/singer/processor/DefaultLogStreamProcessor.java

```diff
@@ -241,7 +247,8 @@ public long processLogStream() throws LogStreamProcessorException {
 
   /**
    * If the decider is not set, this method will return true.
-   * If a decider is set, only return false when the decider's value is 0.
+   * If a decider is set, return false when the decider's value is 0
+   * and disable decider's (if exists) value is 100.
```

Comment:
I think this went back to square one 😂
My initial comment was it should be `or` not `and` here:
```
decider's value is 0 "and" disable decider's
```

Reply:
oh oops thanks for catching that, I definitely ctrl + z'ed too hard on this one and my english is not englishing 😂